sdk/core/System.ClientModel/tests/TestFramework/Mocks/MockPipelineTransport.cs

```diff
@@ -119,20 +143,22 @@ public void SetResponse(int status)
 
     private class TransportRequest : PipelineRequest
     {
+        private string _method;
         private Uri? _uri;
         private readonly PipelineRequestHeaders _headers;
 
         public TransportRequest()
         {
             _headers = new MockRequestHeaders();
             _uri = new Uri("https://www.example.com");
+            _method = "GET";
         }
 
         public override void Dispose() { }
 
         protected override BinaryContent? ContentCore
         {
-            get => throw new NotImplementedException();
+            get => null;
```

Comment:
Why null?  Should there be a way to set the content?  (This may be implemented in another mock request ... feel free to copy...)